trieste/models/model_interfaces.py

```diff
@@ -71,6 +71,23 @@ def sample(self, query_points: TensorType, num_samples: int) -> TensorType:
         """
         raise NotImplementedError
 
+    def predict_y(self, query_points: TensorType) -> tuple[TensorType, TensorType]:
+        """
+        Return the mean and variance of the independent marginal distributions at each point in
+        ``query_points`` for the actual observations, including noise contributions.
```

Comment:
I dont think you need the word actual, either here or line 87. It jars on me for some reason, maybe its just not standard terminology. Observations are observations...